data/campaigns/Dead_Water/scenarios/09_The_Mage.cfg

```diff
@@ -365,30 +395,168 @@
         [/if]
     [/event]
     # ***** END BATS SECTION********************END BATS SECTION*****
-
     [event]
         name=moveto
         [filter]
             side=1
-            x=8
-            y=5
+            [and]
+                race=bats
+                [or]
+                    variation=bat
+                [/or]
+            [/and]
+            x,y=8,5
         [/filter]
+        [message]
+            speaker=unit
+            message= _ "Neep, neep, neep!"
+        [/message]
+        [unstore_unit]
+            variable=Caladon_stored
+            find_vacant=yes
+        [/unstore_unit]
+        [message]
+            speaker=Caladon
+            message= _ "Another of these ACCURSED bats! BEGONE!"
+        [/message]
+        [animate_unit]
+            [filter]
+                id=Caladon
+            [/filter]
+            flag = attack
+            [primary_attack]
+                range=ranged
+            [/primary_attack]
+            [facing]
+                x,y=$x1,$y1
+            [/facing]
+            hits=no
+        [/animate_unit]
+        [sound]
+            name=bat-flapping.wav
+        [/sound]
+        [floating_text]
+            x,y=$x1,$y1
+            text= "<span color='red'>" + _ "SHRIEK!" + "</span>"
+        [/floating_text]
+        [move_unit]
+            x,y=$x1,$y1
+            to_x=10
+            to_y=6
+        [/move_unit]
+        [move_unit]
+            id=Caladon
+            to_x=8
+            to_y=5
+        [/move_unit]
+        [delay]
+            time=400
+            accelerate=yes
+        [/delay]
+        [store_unit]
+            [filter]
+                id=Caladon
+            [/filter]
+            variable=Caladon_stored
+            kill=yes
+        [/store_unit]
+        [message]
+            speaker=Cylanna
+            message= _ "Caladon does not seem very happy about the bat infestation." + " " + _ "We should send a merman unit to talk with him."
```

Comment:
I think it might be better to avoid Cylanna saying the second sentence, or change it to something like "He might be more welcoming of one of our people", and clarify in objectives that the player should reach the hut with a Merman unit.